README.md

```diff
@@ -1,39 +1,84 @@
-# FrontendHelpers
+# Grupo AFAL FrontEnd Helpers
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/frontend_helpers`. To experiment with that code, run `bin/console` for an interactive prompt.
+Collection of components and utilities to facilitate developing web applications integrating Ruby helpers to facilitate the generation of HTML templates, Bulma componentes with some customizations and StimulusJS controllers to provide the functionality to those components.
 
-TODO: Delete this and the text above, and describe your gem
+## Ruby/Rails Helpers
 
-## Installation
+### Instalation
 
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'frontend_helpers'
+gem 'frontend_helpers', github: 'Grupo-AFAL/frontend-helpers', branch: 'main'
 ```
 
 And then execute:
 
     $ bundle install
 
-Or install it yourself as:
+### Usage
 
-    $ gem install frontend_helpers
+TODO: Write how to use Ruby helpers
 
-## Usage
-
-TODO: Write usage instructions here
-
-## Development
+### Development
 
 After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
 To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
-## Contributing
+## Javascript Controllers and utilities
+
+### Instalation
+
+```
+yarn add https://github.com/Grupo-AFAL/frontend-helpers.git
+```
+
+### Usage
+
+All controllers have dependency on StimulusJS which is a peerDependency of this package.
+
+```js
+import { Application } from 'stimulus'
+import { DropdownController } from 'frontend-helpers'
+
+const application = Application.start()
+application.register('dropdown', DropdownController)
+```
+
+### Development
+
+The javascript code live under the `javascript` folder.
```

Comment:
Remember to modify, the Code on the index.scss for the node_modules/bulma/bulma.scss for the App Look on the Relative URL "./"   Not on the Absolute Path.